TeamCode/src/main/java/org/firstinspires/ftc/teamcode/CenterStageManualV3.java

```diff
@@ -0,0 +1,268 @@
+package org.firstinspires.ftc.teamcode;
```

Comment:
Do a small write up on the top of the file as documentation :  Stating what are you trying to achieve by this version 3 of the implementation